src/main/java/appeng/fluids/items/FluidDummyItem.java

```diff
@@ -75,6 +78,47 @@ public void setFluidStack( ItemStack is, FluidStack fs )
 		}
 	}
 
+	public static ItemStack createStackfromFluidContainer (final ItemStack stack)
+	{
+		if (stack.isEmpty())
+		{
+			return ItemStack.EMPTY;
+		}		 
+		final IFluidHandlerItem fluidHandler = FluidUtil.getFluidHandler(stack);
+		if (fluidHandler != null)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)